test/k8s_logging_tests/test_config_logging.py

```diff
@@ -38,139 +48,145 @@ def test_cluster_name(setup, test_input, expected):
-                         len(events))
+                                      url=setup["splunkd_url"],
+                                      user=setup["splunk_user"],
+                                      query=["search {0}".format(
+                                          search_query)],
+                                      password=setup["splunk_password"])
+    logger.info("Splunk received %s events in the last minute",
+                len(events))
     assert len(events) >= expected
 
 
 @pytest.mark.parametrize("test_input,expected", [
-    ("test-namespace-routing", 1),
-    ("kube-system", 1),
-    ("kube-public", 1)
+    ("pod-w-index-wo-ns-index", 1),
+    ("pod-wo-index-w-ns-index", 1),
+    ("pod-w-index-w-ns-index", 1)
 ])
-def test_namespace_routing(setup, test_input, expected):
-    '''
-    Test namespace routing feature. This test will create an index in Splunk with name as the namespace, start a pod to generate some logs,
-    then check if the logs are indexed as events in Splunk. The test handles the cleanup of the index and its events.
-    '''
-    # Splunk index and namespace are assumed to be the same
-    index = test_input
-    namespace = test_input
-
-    # Handle special cases of default namespaces kube-system and kube-public
-    if test_input == "kube-system" or test_input == "kube-public":
-        search_query = "index={0}".format(test_input)
-        events = check_events_from_splunk(index=index,
-                                          start_time="-1h@h",
-                                          url=setup["splunkd_url"],
-                                          user=setup["splunk_user"],
-                                          query=["search {0}".format(search_query)],
-                                          password=setup["splunk_password"])
-        logging.getLogger().info("Received {0} events in the index {1}".format(len(events), index))
-        assert len(events) >= 0
-        pytest.skip("Test successful, skipping rest of the test for special cases")
-
-    # Initialize kubernetes python client
-    config.load_kube_config()
-    v1 = client.CoreV1Api()
-    found = False
-
-    # Search for namespace
-    for ns in v1.list_namespace().items:
-        if test_input == ns.metadata.name:
-            found = True
-
-    # Create namespace
-    if not found:
-        logging.getLogger().info("creating namespace")
-        try:
-            v1.create_namespace(client.V1Namespace(metadata=client.V1ObjectMeta(name=test_input)))
-        except ApiException as e:
-            logging.getLogger().info("Exception when calling CoreV1Api create_namespace: {0}".format(e))
-
-    search_query = "index={0} | delete".format(test_input)
-    events = check_events_from_splunk(index=index,
-                                      start_time="-1h@h",
+def test_label_collection(setup, test_input, expected):
+    '''
+    Test that user specified cluster-name is attached as a metadata to all the logs
```

Comment:
"Test that user specified cluster-name" could be "Test that user specified labels/annotations"